feathr_project/feathr/_synapse_submission.py

```diff
@@ -75,7 +75,7 @@ def submit_feathr_job(self, job_name: str, main_jar_path: str,  main_class_name:
             job_tags (str): tags of the job, for exmaple you might want to put your user ID, or a tag with a certain information
             configuration (Dict[str, str]): Additional configs for the spark job
         """
-
+        assert main_jar_path, 'main_jar_path should not be none but it is none.'
```

Comment:
It seems that main_jar_path now has an invalid default parameter?

Reply:
this is might not be clear enough as itself is a invalid param. and it doesn't tell why it's invalid.